crates/edr_napi/src/spec.rs

```diff
@@ -85,6 +86,39 @@ pub trait SyncNapiSpec:
 impl SyncNapiSpec for L1ChainSpec {
     const CHAIN_TYPE: &'static str = "L1";
 
+    fn cast_response(
+        response: Result<ResponseWithTraces<Self>, ProviderError<Self>>,
+    ) -> napi::Result<Response> {
+        let response = jsonrpc::ResponseData::from(response.map(|response| response.result));
+
+        serde_json::to_string(&response)
```

Comment:
This code duplication will be removed in https://github.com/NomicFoundation/edr/pull/659/files#diff-bd82f1c79ac0cc9d088b3262a78277433ed1667e6f519c2ac84215247a6fcf61